enet server test/enet server test.cpp

```diff
@@ -2147,6 +2148,9 @@ int _tmain(int argc, _TCHAR* argv[])
 			}
 
 			event.peer->data = new PlayerInfo;
+			/* Get the string ip from peer */
+			char clientConnection[32];
```

Comment:
16 bytes should be enough.
```suggestion
			char clientConnection[16];
```